internal/buffer/save.go

```diff
@@ -303,6 +303,13 @@ func (b *Buffer) saveToFile(filename string, withSudo bool, autoSave bool) error
 	result := <-saveResponseChan
 	err = result.err
 	if err != nil {
+		if errors.Is(err, fs.ErrPermission) {
+			return errors.Unwrap(err)
```

Comment:
Why is `fs.ErrPermission` a special case? and why unwrap it?

Reply:
Hm, long time ago already...
Looks like it is a leftover before the file permission was checked with `openFile()`.
So there doesn't seem to be a reason to keep it any longer.